parachain/src/chain_spec.rs

```diff
@@ -390,6 +390,7 @@ fn testnet_genesis(
         },
         issue: IssueConfig { issue_period: DAYS },
         redeem: RedeemConfig {
+            redeem_transaction_size: 600,
```

Comment:
Either leave this undefined or import the constant calculation (suggested above) here.